docs/topics/RPC.md

```diff
@@ -13,7 +13,7 @@ All Discord clients have an RPC server running on localhost that allows control
 
 ## Restrictions
 
-For connections to the RPC server, a [whitelist](#DOCS_TOPICS_RPC/authorize) is used to restrict access while you're still developing. You can invite up to 50 people to your whitelist.
+For connections to the RPC server, an [allowlist](#DOCS_TOPICS_RPC/authorize) is used to restrict access while you're still developing. You can invite up to 50 people to your allowlist.
```

Comment:
since the dashboard also refers to them to testers it might make more sense to write this page in the trend of "while developing this features is restricted to testers of your app, you can add 50 testers to your project in the dashboard"? same for below